webapp/templates/webapp/project/detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends '../__base.html' %}
+{% load i18n %}
+
+{% block body %}
+<h3>{{ project.project_name }}</h3>
+<a href="{% url 'project-edit' project.id %}" class="btn btn-secondary">{% trans 'Editar' %}</a>
+<a href="{% url 'project-list' %}" class="btn btn-link">{% trans 'Volver' %}</a>
++<br>
```

Comment:
Ten en cuenta que este archivo es `webapp/templates/webapp/project/detail.html`. En este archivo de hecho sobra toda la parte del `for`